trigger/testTimer/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+title: CSV Timer
+
+-->
+# Timer Trigger
+This trigger provides your flogo application the ability to get data from a CSV file after a particular interval for an action 
+
+## Installation
+
+```bash
+flogo install github.com/skothari-tibco/csvtrigger
+```
+
+## Metadata
```

Comment:
Remove this metadata entry, follow something closer to https://github.com/project-flogo/contrib/tree/master/trigger/rest . README.md